### ISSUE TITLE
Decreasing Docker image size by cleaning yum cache?

### ISSUE THREAD

**Issue by JulianHn** [ome/omero-server-docker#65]
Hey,

inspired by the discussion here https://github.com/ome/omero-web-docker/issues/67, I checked the Dockerfile for the OMERO.server Docker as well and it is possible to decrease the image size significantly by cleaning the yum caches after installing.

Docker diff:
```
--- a/Dockerfile
+++ b/Dockerfile
@@ -9,15 +9,20 @@ WORKDIR /opt/setup
 ADD playbook.yml requirements.yml /opt/setup/
 
 RUN yum -y install epel-release \
     && yum -y install ansible sudo ca-certificates \
-    && ansible-galaxy install -p /opt/setup/roles -r requirements.yml
+    && ansible-galaxy install -p /opt/setup/roles -r requirements.yml \
+    && yum -y clean all \
+    && rm -fr /var/cache
 
 ARG OMERO_VERSION=5.6.4
 ARG OMEGO_ADDITIONAL_ARGS=
 ENV OMERODIR=/opt/omero/server/OMERO.server/
 RUN ansible-playbook playbook.yml \
     -e omero_server_release=$OMERO_VERSION \
-    -e omero_server_omego_additional_args="$OMEGO_ADDITIONAL_ARGS"
+    -e omero_server_omego_additional_args="$OMEGO_ADDITIONAL_ARGS" \
+    && yum -y clean all \
+    && rm -fr /var/cache
 
 RUN curl -L -o /usr/local/bin/dumb-init \
     https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64 && \
```

This leads to the following image sizes (updated has an additional `yum update -y` in the first RUN command.

| Options | Image Size |
| Current | 2.34 GB |
| Cleaned | 1.63 GB |
| Cleaned & Update | 1.73 GB |

This would be a 700MB or almost 30% decrease in image size or roughly 25% if the base CentOS image is updated during the build.

// Julian

**Comment by joshmoore** [comment]
:+1: @JulianHn: thanks for looking into that. Do you want to open a PR?

**Comment by JulianHn** [comment]
@joshmoore Sure. What is the verdict on performing a `yum update`?

**Comment by JulianHn** [comment]
Implemented into the codebase with #69.
